OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -194,6 +209,10 @@ public class PodCommsSession {
         
         if blocksToSend.contains(where: { $0 as? NonceResyncableMessageBlock != nil }) {
             podState.advanceToNextNonce()
+            let padNonceMessages = false // edit to pad non deactivatePod nonce messages
+            if padNonceMessages && messageBlocks[0].blockType != .deactivatePod {
```

Comment:
I don't think debug/dev code should linger in the code, but I think we should talk through whether we want to enable this for Loop in general.  Which nonce commands end up getting padded?

Reply:
The $08, $11, $1F and most $19 commands would end up getting padded when this code is enabled. I'm happy to talk about whether the nonce command padding should be enabled for Loop in general any time! My vote at this time would be not to bother, but to have this code available in case anyone wants to do some testing with and without nonce command padding enabled.

Reply:
Hmm, the only one that I could see being a significant benefit is the cancel delivery ($1F) command. But yeah, out of all the commands, bolus and temp basal are by far the most important to have this feature on, and those already have the feature of two packets. I am fine leaving this branch out there for future reference, and I think that we do have some other options in terms of recovering from an uncertain delivery state. Some of the code in this latest merge will support automatic recovery once the PumpManager side of it is implemented in OmniKit.